### PR TITLE
data:MutiatBash

### PR DESCRIPTION
**Pull request by MutiatBash**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/MutiatBash.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  "url":"https://twitter.com/dev_mutiah",
+  "icon":"FaTwitter",
+  "color":"#00ACEE",
+},
+     {
+  "group":"Socials",
+  "name":"LinkedIn:",
+  "url":"https://www.linkedin.com/in/mutiat-bashua",
+  "icon":"FaLinkedIn",
+  "color":"#00ACEE",
```
Comment (sital002) [inline data/MutiatBash.json:17]:
```suggestion
  "color":"#00ACEE"
```
The trailing comma is not required

File: data/MutiatBash.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+  "name":"Mutiat Bashua",
+  "bio":"Frontend Developer | Mobile App Developer | [Github Profile](https://github.com/MutiatBash)",
+  "links":[
+    {
+  "group":"Socials",
+  "name":"Twitter:",
+  "url":"https://twitter.com/dev_mutiah",
+  "icon":"FaTwitter",
+  "color":"#00ACEE",
```
Comment (sital002) [inline data/MutiatBash.json:10]:
```suggestion
  "color":"#00ACEE"
```
The trailing comma isnot required

**Review by sital002 — CHANGES_REQUESTED** [review]
Please make the necessary changes to get your profile merged.

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**